src/main/kotlin/graphics/scenery/volumes/OrthoView.kt

```diff
@@ -22,8 +23,9 @@ import kotlin.concurrent.thread
  * [Volume.slicingMode] should be set to [Volume.SlicingMode.None]
  *
  * @param key The input key for optional plane movement behavior. See Input Docs for format.
+ * @param hub only needed when [key] is set
  */
-fun createOrthoView(volume: Volume, key: String? = null) {
+fun createOrthoView(volume: Volume, key: String? = null, hub: Hub?) {
```

Comment:
Why does orthoview need a hub now?

Reply:
To add a addOrthoViewDragBehavior. This is an optional convinience feature. Hub can be null if the feature is not desired. Previously the function got the hub from volume which loses its hub property in this PR.

Reply:
Alright for the moment, we'd probably have to think of a better way down the road.